Babel-runtime not found error

When a create a new project from scratch with swan setup, the i get following error:

resolve failed:  { Error: Cannot find module 'babel-runtime'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.requireRelative.resolve (/home/bzd/swan-example/swan-example-client/node_modules/require-relative/index.js:30:17)
    at resolve (/home/bzd/swan-example/swan-example-client/node_modules/modify-babel-preset/lib/serialize.js:26:26)
    at findAndRemove (/home/bzd/swan-example/swan-example-client/node_modules/modify-babel-preset/lib/serialize.js:83:11)
    at /home/bzd/swan-example/swan-example-client/node_modules/modify-babel-preset/lib/serialize.js:126:13
    at Array.map (native)
    at loadPreset (/home/bzd/swan-example/swan-example-client/node_modules/modify-babel-preset/lib/serialize.js:118:29)
    at module.exports (/home/bzd/swan-example/swan-example-client/node_modules/modify-babel-preset/index.js:95:19)
    at Object.<anonymous> (/home/bzd/swan-example/swan-example-client/node_modules/babel-preset-es2015-loose/index.js:5:18)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at /home/bzd/swan-example/swan-example-client/node_modules/babel-core/lib/transformation/file/options/option-manager.js:302:17
    at Array.map (native)
    at OptionManager.resolvePresets (/home/bzd/swan-example/swan-example-client/node_modules/babel-core/lib/transformation/file/options/option-manager.js:270:20)
    at OptionManager.mergePresets (/home/bzd/swan-example/swan-example-client/node_modules/babel-core/lib/transformation/file/options/option-manager.js:259:10)
    at OptionManager.mergeOptions (/home/bzd/swan-example/swan-example-client/node_modules/babel-core/lib/transformation/file/options/option-manager.js:244:14)
    at OptionManager.init (/home/bzd/swan-example/swan-example-client/node_modules/babel-core/lib/transformation/file/options/option-manager.js:374:12)
    at File.initOptions (/home/bzd/swan-example/swan-example-client/node_modules/babel-core/lib/transformation/file/index.js:216:65)
    at new File (/home/bzd/swan-example/swan-example-client/node_modules/babel-core/lib/transformation/file/index.js:139:24)
    at Pipeline.transform (/home/bzd/swan-example/swan-example-client/node_modules/babel-core/lib/transformation/pipeline.js:46:16)
    at DestroyableTransform._transform (/home/bzd/swan-example/swan-example-client/node_modules/gulp-babel/index.js:37:20)
    at DestroyableTransform.Transform._read (/home/bzd/swan-example/swan-example-client/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:159:10)
    at DestroyableTransform.Transform._write (/home/bzd/swan-example/swan-example-client/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:147:83)
    at doWrite (/home/bzd/swan-example/swan-example-client/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:347:64)
    at writeOrBuffer (/home/bzd/swan-example/swan-example-client/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:336:5)
    at DestroyableTransform.Writable.write (/home/bzd/swan-example/swan-example-client/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:274:11)
    at DestroyableTransform.ondata (/home/bzd/swan-example/swan-example-client/node_modules/through2/node_modules/readable-stream/lib/_stream_readable.js:546:20)
    at emitOne (events.js:96:13)
    at DestroyableTransform.emit (events.js:188:7) code: 'MODULE_NOT_FOUND' } babel-runtime
/home/bzd/swan-example/swan-example-client/node_modules/gulp-util/lib/PluginError.js:45
        self[prop] = options.error[prop];


When I downgrade  "babel-preset-es2015-loose": "^8.0.0" to     "babel-preset-es2015-loose": "^7.0.0"
the problem is gone.

But it doesn't seem to be a good solution to downgrade the package.

--- a/build/babel-options.js
+++ b/build/babel-options.js
@@ -12,7 +12,7 @@ exports.base = function() {
     comments: false,
     compact: false,
     code: true,
-    presets: [ 'es2015-loose', 'stage-1'],
+    presets: [ ['es2015', {loose: true}], 'stage-1' ],
     plugins: [
       'syntax-flow',
       'transform-decorators-legacy',

--- a/package.json
+++ b/package.json
@@ -65,7 +65,7 @@
     "object.assign": "^4.0.4",
     "require-dir": "^0.3.0",
     "run-sequence": "^1.2.2",
-    "spoonx-tools": "^1.0.0-0",
+    "spoonx-tools": "^1.1.1",
     "vinyl-paths": "^2.1.0",
     "yargs": "^6.0.0"
   },

